### PR TITLE
Create-New-User screen

### LINKED ISSUE #31: Giving too much info at failed login
We're giving too much information when checking a user when they're trying to log in. We should change line 92 in the user facade to just return a promise with the value `false` so the user doesn't know if the username or the password was incorrect (or both).

### PR DESCRIPTION
**Pull request by MrHardcode**
This is a PR for a "I don't have a user so I want to create one"-screen
Also closes #31 
__Tasks__

* [x] Create first version that adds a dummy user
* [x] Connect input fields to state so that users can type in their own info and submit this
* [ ] Hook up the component with `app.js` so users can navigate to the **Create User**-screen easily
* [x] Deny login after 3 attempts for the next hour or so, and longer if the attempts continue to fail.
* [x] Password control
  * [x] should be at least length X (min 10 characters)
  * [x] should contain letters and numbers and special characters 
  * [x] check against list of known passwords (if possible and if time permits it)

### CODE REVIEW

**Comment by MalteMagnussen** [comment]
Er det en "Create a User" screen, eller er det "I want to create a User-Screen"? 😅

File: Backend/src/resolverMap.ts [hunk 1]
```diff
@@ -53,11 +53,11 @@ const resolverMap: IResolvers = {
...
-      if (!validateEmail(email)) {
-        throw new UserInputError("Email Argument invalid", {
-          invalidArgs: "email",
-        });
-      }
+      // if (!validateEmail(email)) {
+      //   throw new UserInputError("Email Argument invalid", {
+      //     invalidArgs: "email",
+      //   });
+      // }
```
Comment (MrHardcode) [inline Backend/src/resolverMap.ts:66]: I don't know why but this check is so aggressive that it basically doesn't accept any emails

Comment (MalteMagnussen) [reply Backend/src/resolverMap.ts:66]: Okay. Either we just completely remove it then, like you have here, or we need to find an alternative solution.

Comment (MrHardcode) [reply Backend/src/resolverMap.ts:66]: Maybe we need another regular expression

**Comment by MrHardcode** [comment]
I have updated the description to make it more clear what exactly this PR contains

**Comment by MrHardcode** [comment]
I made it so that users are only allowed to try to log in once every second (interval can be changed easily) and if they try then their IP will be blocked from attempting to log in for 5 seconds (period can easily be adjusted). Is this acceptable or should it be changed?

**Comment by MalteMagnussen** [comment]
Is this ready for a review tomorrow morning? :)

**Review by Castau — APPROVED** [review]

File: Backend/src/server.ts [hunk 2]
```diff
@@ -10,24 +10,52 @@ import { ApiError } from "./customErrors/apiError";
...
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
 app.use(passport.initialize());
 
-app.use(requestLogger);
 //The regular logger needs to be before the router
-app.use(requestLogger);
+// app.use(requestLogger);
```
Comment (MalteMagnussen) [inline Backend/src/server.ts]: Bliver den slet ikke brugt nu så?

**Review by MalteMagnussen — APPROVED** [review]

File: Backend/src/server.ts (hunk 2 above)
Comment (MrHardcode) [reply Backend/src/server.ts]: I only removed it because I was testing something else - forgot to remove the comment. We're still supposed to use it

**Pull request merged**